release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -2892,6 +2892,42 @@ This section will continue to be updated over time to provide notes on enhanceme
 For any {product-title} release, always review the instructions on xref:../updating/updating_a_cluster/updating-cluster-web-console.adoc#updating-cluster-web-console[updating your cluster] properly.
 ====
 
+// 4.17.31
+[id="ocp-4-17-31_{context}"]
+=== RHBA-2025:8108 - {product-title} {product-version}.31 bug fix and security update advisory
```

Comment:
I don't think this would be a security update because it doesn't have an RHSA. Also, someone in the RN team needs set their editor to trim whitespace. Good job setting yours and catching all this. @dfitzmau